src/Domain.Linnapps/Products/SalesArticle.cs

```diff
@@ -45,6 +45,8 @@ public class SalesArticle
 
         public ICollection<SaHoldStory> HoldStories { get; set; }
 
+        public int TariffId { get; set; }
```

Comment:
I'd put a comment here to say we're deliberately not mapping this properly as it is set in the new products app